superset/assets/visualizations/treemap.css

```diff
@@ -6,10 +6,10 @@ text {
   font-weight: bold;
 }
 
-rect {
+/*rect {
```

Comment:
if not delete,  the multiBarHorizontalChart can't display, like that:
![g](https://cloud.githubusercontent.com/assets/19641641/20922312/ab64d598-bbe2-11e6-9580-4c31d6a786f4.jpg)